collections/nemo_nlp/nemo_nlp/utils/callbacks/glue.py

```diff
@@ -80,9 +77,9 @@ def eval_epochs_done_callback(global_vars, output_dir, task_name):
     if preds.shape[0] > 21:
         i = random.randint(0, preds.shape[0] - 21)
 
-    logger.info("Task name: %s" % task_name.upper())
-    logger.info("Sampled preds: [%s]" % list2str(preds[i:i+20]))
-    logger.info("Sampled labels: [%s]" % list2str(labels[i:i+20]))
+    nemo.logging.info("Task name: %s" % task_name.upper())
```

Comment:
debug?

Reply:
I would prefer logging examples during evaluation. It helps to see if your model is training.
Therefore, I would argue to keep it at info

Reply:
ok, makes sense